ring_tx.h

```diff
@@ -30,12 +30,24 @@ static inline void kernel_may_pull_from_tx(struct tpacket2_hdr *hdr)
 
 static inline int pull_and_flush_tx_ring(int sock)
 {
-	return sendto(sock, NULL, 0, MSG_DONTWAIT, NULL, 0);
+	int ret = -1;
+
+	do {
+		ret = sendto(sock, NULL, 0, MSG_DONTWAIT, NULL, 0);
+	} while(ret == -1 && errno == EINTR);
```

Comment:
nit: space between `while` and opening brace like in the rest of the netsniff-ng code base:

```suggestion
	} while (ret == -1 && errno == EINTR);
```